.github/workflows/sync-critical-path.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: find . -name "*.apk"  -exec mv '{}' apk/release.apk \;
 
       - name: Sync Flows
-        uses: mobile-dev-inc/action-maestro-cloud@v1.6.0
+        uses: mobile-dev-inc/action-maestro-cloud@v1.8.1
```

Comment:
I know it might not be part of these PR, but we have multiple maestro action versions, 1.4.1, 1.6.0 and now also 1.8.1.
Can we (as part of this PR) update all yml files to use the same maestro version?

Reply:
Sounds good @aitorvs , I’ll take care of it